controller/cache.ml

```diff
@@ -22,7 +22,7 @@ let memo_cache_file = ".coq-lsp.cache"
 
 let memo_save_to_disk () =
   try
-    Fleche.Memo.save_to_disk ~file:memo_cache_file;
+    (* Fleche.Memo.save_to_disk ~file:memo_cache_file; *)
```

Comment:
Did you mean to comment these out?

Reply:
Yes, the stuff in Memo was really dead code so I've removed it, I'd like to keep these ones as stubs for now tho.